pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -2,12 +2,11 @@
 = Description of the {ProductShortName} configuration file
 
 [role="_abstract"]
-The configuration file, named `pantheon2.yml`, resides in the root directory of your documentation repository. You use this file to designate a Git repository as a source that {ProductShortName} can use to import content. This configuration file provides the information necessary for {ProductShortName} to interact with files in the repository.
+The configuration file, named `{ProductShortName}2.yml`, resides in the root directory of your documentation repository. You use this file to designate a Git repository as a source that {ProductShortName} can use to import content. This configuration file provides the information necessary for {ProductShortName} to interact with files in the repository.
```

Comment:
I would not use this attribute here. The casing is wrong, and it wouldn't be the case that if someone else gave this file another name it would include a 2. Just leave this as pantheon2.yml.